### DISCUSSION TITLE
Cluster friendly

### DISCUSSION THREAD

**Discussion by adamalexandru4** [kagkarlsson/db-scheduler#411] [Q&A] [answered]
I don't think that cluster has enough details in order to know the limitation of the library.
Cluster of server instances ? Cluster of schedulers on the same app ? Cluster of .. ?

**Comment by kagkarlsson** [comment, marked as answer]
Cluster of Scheduler-instances. They can run in different JVMs or multiple in the same JVM. They must have the same Task-set.
